src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/ptv/PtvTranslatorTest.java

```diff
@@ -94,10 +99,37 @@ public void testMerged() throws IOException, JSONException {
     }
   }
   
-  private void assertServiceHoursEqual(List<ServiceHour> expected, List<ServiceHour> actual) throws IOException, JSONException {
-    String expectedString = getObjectMapper().writeValueAsString(expected);
-    String actualString = getObjectMapper().writeValueAsString(actual);
-    JSONAssert.assertEquals("ServiceHours are not equal", expectedString, actualString, false);
+  private void assertServiceHoursEqual(List<ServiceHour> expectedServiceHours, List<ServiceHour> actualServiceHours) throws IOException, JSONException {
+    assertEquals(expectedServiceHours.size(), actualServiceHours.size());
+    for (int i = 0; i < expectedServiceHours.size(); i++) {
+      ServiceHour expectedServiceHour = expectedServiceHours.get(i);
+      ServiceHour actualServiceHour = actualServiceHours.get(i);
+      assertEquals(expectedServiceHour.getServiceHourType(), actualServiceHour.getServiceHourType());
+      assertInstantsMatch(expectedServiceHour.getValidFrom(), actualServiceHour.getValidFrom());
+      assertInstantsMatch(expectedServiceHour.getValidTo(), actualServiceHour.getValidTo());
+      assertEquals(expectedServiceHour.getIsClosed(), actualServiceHour.getIsClosed());
+      assertEquals(expectedServiceHour.getValidForNow(), actualServiceHour.getValidForNow());
+      assertJSONEquals(expectedServiceHour.getAdditionalInformation(), actualServiceHour.getAdditionalInformation());
+      assertJSONEquals(expectedServiceHour.getOpeningHour(), actualServiceHour.getOpeningHour());
+    }
+  }
+
+  private void assertJSONEquals(Object expected, Object actual) throws JsonProcessingException, JSONException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)